documentation/examples/11-easing/00-easing/Controls.svelte

```diff
@@ -74,26 +74,28 @@
 	ul {
 		list-style: none;
 		padding: 0;
-		display: flex;
-		flex-direction: column;
-		align-items: flex-start;
+		display: grid;
+		grid-template-columns: 1fr;
+		gap: 3px;
 		font-size: 18px;
 	}
 
 	li {
-		padding: 5px 10px;
-		background: #eee;
-		border-radius: 2px;
-		margin: 3px 0;
-		cursor: pointer;
+		display: grid;
 	}
 
 	li:hover {
 		background: #676778;
 		color: white;
 	}
 
-	.selected {
+	button {
+		border: none;
+		border-radius: 2px;
+		padding: 2px;
+	}
+
+	.selected > button {
 		background: #ff3e00;
 		color: white;
```

Comment:
I have a little bit of a hard time reading the selected one. I think we can address it by making it bold
```suggestion
		color: white;
		font-weight: bold;
```

Reply:
Is it better with the bold?
![image](https://github.com/sveltejs/svelte/assets/50981692/a32217be-6dae-4a36-a506-e55733812b36)

Looks more clumped together and less clear imo

Reply:
Here's what I see without it. It's rather hard to read for me. I'm not sure what how you mean that bold would make them look clumped together as it doesn't affect the spacing between them

![Screenshot from 2023-08-25 07-17-57](https://github.com/sveltejs/svelte/assets/322311/9c28b2c0-3551-470f-8b2b-a371b5ca5879)

Reply:
Ahh, they look a lot thinner on yours than they do on my screen. Bold is fine then.